lib/taste_tester/tunnel.rb

```diff
@@ -44,15 +44,13 @@ def run
     end
 
     def cmd
-      if TasteTester::Config.user != 'root'
-        pid = '$$'
-      else
-        pid = '\\$\\$'
```

Comment:
It doesn't work with the backslashes though.

How do I test all the cases, I'm not running taste-tester as root. I'm not sure why it requires root permissions to begin with.